shared/features/contributors/contributor-profile-extended/contributor-profile-extended.tsx

```diff
@@ -81,19 +80,7 @@ export function ContributorProfileExtended({ user, headerProps, footerContent }:
         <div className={"flex w-full flex-row items-center justify-between gap-1"}>
           <div className={"flex flex-1 flex-row flex-wrap items-center gap-md"}>
             {contributor.contacts.map(contact => (
-              <Button
-                key={contact.contact}
-                iconOnly={true}
-                as={"a"}
-                size={"sm"}
-                variant={"secondary"}
-                htmlProps={{
-                  href: contact.contact,
-                  target: "_blank",
-                  rel: "noreferrer",
-                }}
-                startContent={<SocialIconLink url={contact.contact} />}
-              />
+              <SocialLinkTag key={contact.contact} url={contact.contact} channel={contact.channel} />
```

Comment:
No value is passed

Reply:
we only need the url here since the tag is icon only